.cargo/config

```diff
@@ -1,5 +1,27 @@
 [alias]
 xtask = "run --package xtask --"
+ruma-clippy = """\
+  clippy -p ruma --workspace --all-targets --all-features --quiet -- \
```

Comment:
It's weid that `-p` and `--workspace` even work together. We shouldn't specify a package when all packages from the workspace are being checked though.

```suggestion
  clippy --workspace --all-targets --all-features --quiet -- \
```

Reply:
Wtf!! after fixing that I got another 60 clippy warnings, all different kinds why is clippy so non-deterministic seemingly!